docs/application/dotnet/get-started/index.md

```diff
@@ -0,0 +1,121 @@
+
+For more information, see the [Visual Studio Tools for Tizen](../../vstools/index.md) guides.
+
+## Create your first app
+
+To create your first Tizen .NET application, you can select from the following types:
+
+<div class="row cards-container-infra">
+    <div class="col col-6">
+        <a href="wearable/first-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">Wearable .NET</p>
+                <p class="card-text">An application that is targeted to run on smartwatches. It can be optimized for small screens.</p>
+            </div>
+        </a>
+    </div>
+    <div class="col col-6">
+        <a href="wearable/first-xaml-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">Wearable XAML</p>
+                <p class="card-text">An application that allows you to use XAML to define the visual contents of a page and work together with a C# code-behind file on smartwatches.</p>
+            </div>
+        </a>
+    </div>
+    <div class="col col-6">
+        <a href="mobile/first-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">Mobile .NET</p>
+                <p class="card-text">An application that is targeted to run on smartphones. It can take advantage of the functionalities available on smartphones.</p>
+            </div>
+        </a>
+    </div>
+    <div class="col col-6">
+        <a href="tv/first-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">TV .NET</p>
+                <p class="card-text">An application that is targeted to run on smart TVs. It can be optimized for large screens and remote controls.</p>
+            </div>
+        </a>
+    </div>
+    <div class="col col-6">
+        <a href="iot/first-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">IoT .NET</p>
+                <p class="card-text">An application that is targeted to run on IoT devices. It can be optimized for both headed as well as headless IoT devices.</p>
+            </div>
+        </a>
+    </div>
+    <div class="col col-6">
+        <a href="nui/first-app.md" class="card card-infra card-neutral h-100">
+            <div class="card-body">
+                <p class="h3 card-title">NUI .NET</p>
+                <p class="card-text">An application that is targeted to run on a wide range of Tizen devices such as mobile devices, TVs, and wearables. NUI is a rich GUI library used for creating two-dimensional or three-dimensional applications.</p>
+            </div>
+        </a>
+    </div>
+</div>
+
+## Next steps
```

Comment:
Next Steps is really needed here? As per staging it looks like two headings appearing consecutively.
https://docs1.stg.tizen.org/staging/1387/application/dotnet/get-started/

Reply:
Just for your information : 
Line 71 is in 2nd depth, and line 73 is in 4th depth.
So in the test URL commented in https://github.com/Samsung/tizen-docs/pull/1387#discussion_r610447808,
they are transformed like below : 

![image](https://user-images.githubusercontent.com/9958374/114153155-fff02780-9959-11eb-876e-3bad39d82931.png)

Reply:
"Next steps" text was replaced with "Run on real device"